<?xml version="1.0"?>
<launch>

  <node pkg="openpose_ros_receiver" type="openpose_ros_receiver_broadcaster_node" name="openpose_ros_receiver_broadcaster_node" output="screen">
    <rosparam command="load" file="$(find openpose_ros_receiver)/config/openpose_ros_receiver_zed.yaml"/>
  </node>

</launch>
